APE10.rst

```diff
@@ -0,0 +1,155 @@
+Roadmap for Python 3-only support
+---------------------------------
+
+author: Tom Robitaille
+
+date-created: 2016 July 13
+
+date-last-revised: 2016 July 13
+
+type: Process
+
+status: Draft
+
+Abstract
+--------
+
+The purpose of this APE is to set out a roadmap for future releases of the
+Astropy core package, in particular to identify the process for switching to
+releases that support only Python 3.
+
+Detailed description
+--------------------
+
+A number of scientific Python projects, including IPython, Matplotlib, Sympy,
+and others have now signed the `Python 3 statement
+<https://python3statement.github.io>`_ which states that scientific Python
+packages will drop support for Python 2 no longer than 2020. In fact, the
+IPython developers have now released `IPython 5.0
```

Comment:
Yeah, I think it's fine to just say that the 5.x is the last that will suport py 2.x.  Although you *could* move this below and say it's motivation for *our* roadmap.